src/test/cypress/integration/Login.spec.ts

```diff
@@ -8,22 +7,14 @@ describe('Login page tests', () => {
     it('Logs in via the UI', () => {
         cy.visit('/');
         loginPage.login(user);
-        cy.url()
-            .should('include', '/courses')
-            .then(() => {
-                expect(localStorage.getItem(authTokenKey)).to.not.be.null;
-            });
+        cy.url().should('include', '/courses');
```

Comment:
should we also expect cookies set/not set? `cy.getCookie(...)`

If yes, please adapt other occurrences in cypress tests

Reply:
We should, done [c74b206](https://github.com/ls1intum/Artemis/pull/5723/commits/c74b206090f41fcb7a0eaf20f52d97ee75d21ed2), thanks!